### PR TITLE
Homepage FxA Experiment (Growth Exp 2) 

### PR DESCRIPTION
**Pull request by maxxcrawford**
**To-do:** 
- [x] Confirm analytics (Amplitude, GA) 
- [x] Set Custom Dimension on page load
- [x] ~Create new applicable integration tests~ Revise integration tests
- [x] Add experiment active logic via .ENV variable

### Treatment

**Note: When testing locally, add the variable `EXPERIMENT_ACTIVE=1` to your `.ENV` file.**

1. Go to https://monitor-v2.herokuapp.com/?experimentBranch=vb
1. Click "Get Started" Button
1. Go through FxA Flow 
1. Once completed, user should be on the `/user/dashboard/` page

## Expected Results

**Control:** 
image

**Treatment:** 
_Note - The text is still being worked on._

image

### CODE REVIEW

**Comment by jrbenny35** [comment]
You should update your branch for the fix

File: controllers/oauth.js [hunk 1]
```diff
@@ -90,12 +111,11 @@ async function confirmed(req, res, next, client = FxAOAuthClient) {
...
     );
     req.session.user = verifiedSubscriber;
-
-    return res.redirect("/user/dashboard");
+    return res.redirect(returnURL.pathname + returnURL.search);
   }
   // Update existing user's FxA data
   await DB._updateFxAData(existingUser, fxaUser.accessToken, fxaUser.refreshToken, fxaProfileData);
-  return res.redirect("/user/dashboard");
+    res.redirect(returnURL.pathname + returnURL.search);
```
Comment (maxxcrawford) [inline controllers/oauth.js:118]: Should this have a `return`?

File: public/js/monitor.js [hunk 2]
```diff
@@ -36,16 +36,26 @@ function isValidEmail(val) {
...
   url = getFxaUtms(url);
 
-  ["flowId", "flowBeginTime", "entrypoint"].forEach(key => {
+  ["flowId", "flowBeginTime", "entrypoint", "entrypoint_experiment", "entrypoint_variation", "form_type"].forEach(key => {
     if (el.dataset[key]) {
       url.searchParams.append(key, encodeURIComponent(el.dataset[key]));
     }
   });
 
+  if (el.dataset.entrypoint === "fx-monitor-create-account-blue-btn") {
```
Comment (maxxcrawford) [inline public/js/monitor.js:49]: Include alternate entry point

File: controllers/oauth.js (hunk 1 above)
Comment (maxxcrawford) [reply controllers/oauth.js:118]: Nevermind. - https://github.com/mozilla/blurts-server/pull/1602/files#diff-075ff2d4bc68f5a42b68daf27075ab5bL97

**Pull request merged**